<!--
 Copyright 2022 Shadow Robot Company Ltd.
 This program is free software: you can redistribute it and/or modify it
 under the terms of the GNU General Public License as published by the Free
 Software Foundation version 2 of the License.
 This program is distributed in the hope that it will be useful, but WITHOUT
 ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
 FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
 more details.
 You should have received a copy of the GNU General Public License along
 with this program. If not, see <http://www.gnu.org/licenses/>.
-->
<launch>
  <arg name="hand_serials_list"/>

  <node name="load_diagnostic_analyzer" pkg="sr_hand_config" type="load_diagnostic_analyzer.py" output="screen">
    <rosparam param="hand_serials_list" subst_value="True">$(arg hand_serials_list)</rosparam>
  </node>
</launch>